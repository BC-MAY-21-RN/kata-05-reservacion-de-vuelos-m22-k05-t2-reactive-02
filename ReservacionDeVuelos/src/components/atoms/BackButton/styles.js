import {StyleSheet, Dimensions} from 'react-native';

const styles = StyleSheet.create({
  containerButton: {marginTop: 9, marginLeft: 30},
});

export default styles;
