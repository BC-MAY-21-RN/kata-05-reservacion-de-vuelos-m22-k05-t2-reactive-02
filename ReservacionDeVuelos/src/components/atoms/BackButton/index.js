import React from 'react';
import {TouchableOpacity} from 'react-native';
import BackSVG from '../../../assets/icons/back.svg';
import styles from './styles';

export default function BackButton({navigation}) {
  return (
    <TouchableOpacity
      style={styles.containerButton}
      onPress={() => navigation.goBack()}>
      <BackSVG />
    </TouchableOpacity>
  );
}
