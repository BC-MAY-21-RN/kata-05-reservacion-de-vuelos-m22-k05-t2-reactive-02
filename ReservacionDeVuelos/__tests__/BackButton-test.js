import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import BackButton from '../src/components/atoms/BackButton';

describe('Tests from BackButton component', () => {
  it('BckButton snapshot', () => {
    const renderBackButton = renderer.create(<BackButton />).toJSON();
    expect(renderBackButton).toMatchSnapshot();
  });
});
