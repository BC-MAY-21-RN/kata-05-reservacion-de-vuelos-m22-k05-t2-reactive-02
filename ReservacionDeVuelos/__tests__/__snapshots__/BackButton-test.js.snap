// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests from BackButton component BckButton snapshot 1`] = `
<View
  accessible={true}
  collapsable={false}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "marginLeft": 30,
      "marginTop": 9,
      "opacity": 1,
    }
  }
>
  < />
</View>
`;
